kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/ListAllReassignmentsActionIntegrationTest.java

```diff
@@ -0,0 +1,73 @@
+package io.confluent.kafkarest.integration.v3;
+
+import static org.junit.Assert.assertEquals;
+
+import io.confluent.kafkarest.Versions;
+import io.confluent.kafkarest.entities.v3.ListAllReassignmentsResponse;
+import io.confluent.kafkarest.entities.v3.ListReassignmentsResponse;
+import io.confluent.kafkarest.entities.v3.ReassignmentData;
+import io.confluent.kafkarest.integration.ClusterTestHarness;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import org.apache.kafka.clients.admin.NewPartitionReassignment;
+import org.apache.kafka.common.TopicPartition;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ListAllReassignmentsActionIntegrationTest extends ClusterTestHarness {
```

Comment:
`RunWith(Junit4.class)`

Reply:
Added it, didn't usually come across the annotation on integration test class .